app/views/giveaways/edit.html.erb

```diff
@@ -1,6 +1,3 @@
-<h1>Edit Giveaway</h1>
-
-<%= render 'form' %>
-
-<%= link_to "BACK", giveaways_path %>
-
+<%= render 'form',
+heading: "Edit Giveaway",
+cancel_path: giveaway_path(@giveaway) %>
```

Comment:
What is the reason for sending cancel_path when rendering the partial?

Reply:
@violeta When rendering from new.html.erb the cancel_path is set to index page, when rendering from edit.html.erb it is set to show page.